src/views/tickets/tickets.jsx

```diff
@@ -285,16 +300,17 @@ export class Tickets extends Component {
                         onSelectAll: (isSelect, rows) => isSelect ? this.handleSelectAll(rows) : this.handleDeselectAll(rows),
                         sort: true,
                         headerColumnStyle: () => ({ width: "5%" }),
-                        nonSelectableStyle: () => ({color: '#999999'})
+                        nonSelectableStyle: () => ({ color: '#999999' })
                       })}
                     />
                   </div>
                 </div>
                 <TicketModal
                   show={this.state.viewedTicket}
                   onClose={this.toggleTicketModal}
-                  ticket={this.state.viewedTicket}>
-                </TicketModal>
+                  ticket={this.state.selectedTicket}
```

Comment:
The modal is not showing up because there is no "selectedTicket" property in state. This should be "this.state.viewedTicket" like before